compose/compose.go

```diff
@@ -30,14 +30,17 @@ type Factory func(config *Config, storage interface{}, strategy interface{}) int
 //  )
 //
 // Compose makes use of interface{} types in order to be able to handle a all types of stores, strategies and handlers.
-func Compose(config *Config, storage interface{}, strategy interface{}, factories ...Factory) fosite.OAuth2Provider {
+func Compose(config *Config, storage interface{}, strategy interface{}, hasher interface{}, factories ...Factory) fosite.OAuth2Provider {
```

Comment:
Good question! I was blindly following the storage implementation. Fixed with latest commit.